src/chester.js

```diff
@@ -119,9 +120,10 @@ module.exports.Chester = (secret: string | Buffer, {
   };
 
   return {
-    _crypter: crypter,
+    _crypter: crypter, // why is this exposed?
     lock,
     unlock,
+    // what is the case for contexts?
```

Comment:
E.g. we want to tie the token to the user agent or to the ip of the user.

Reply:
so only to add additional params to the signature?

Reply:
Correct, additional params on a "lock / unlock" granularity level.